client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -314,6 +357,28 @@ class CancelPurchaseButton extends Component {
 						flowType={ this.getCancellationFlowType() }
 					/>
 				) }
+
+				{ this.shouldHandleMarketplaceSubscriptions() && (
+					<MarketPlaceSubscriptionsDialog
+						isDialogVisible={ this.state.isShowingMarketplaceSubscriptionsDialog }
+						closeDialog={ this.closeDialog }
+						removePlan={ closeDialogAndProceed }
+						planName={ planName }
+						activeSubscriptions={ activeSubscriptions }
+						sectionHeadingText={ translate( 'Cancel %(plan)s', {
+							args: { plan: planName },
+						} ) }
+						primaryButtonText={ translate( 'Continue', {
+							comment:
+								'This button cancels the active plan and all active Marketplace subscriptions on the site',
+						} ) }
+						bodyParagraphText={ translate(
+							'This subscription will be cancelled. It will be removed when your plan expires.',
```

Comment:
```suggestion
							'This subscription will be cancelled. It will be removed when it expires.',
```

Reply:
That new wording is more accurate, thanks for the suggestions!